Project-I-Social-Card/social-card/src/components/FooterComponents/Footer.css

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
good job, this is essentially you making use of styled components which we'll be going over more explicitly 2/21 I believe